qtc_packaging/debian_fremantle/changelog

```diff
@@ -1,3 +1,8 @@
+kodimote (2.0.2-fremantle0) unstable; urgency=low
+  * transition from xbmcremote to kodimote
+
+ -- Michael Zanetti <michael_zanetti@gmx.net>  Sun, 01 Mar 2015 20:00:00 +0200
```

Comment:
I would not bother too much right now with the changelog (and the version). Shortly before actually releasing something, do a special commit just to bump the version and make sure the changelog is up to date. The person that does the release build should then put it's name here.